src/styles/card.css

```diff
@@ -0,0 +1,53 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+}
+
+.card__image {
+  background-image: url(./../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: 160px 134px;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin: 0 0 4px;
+}
+
+.card__code {
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  margin-bottom: 24px;
+}
+
+.card__review {
+  margin: 0;
+}
+
+.card__review--text_small,
+.card__code--text_small {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
```

Comment:
You can define font-size, font-weight, line-height as variables and then you can define typography for this to re-use the code. This comment for all the typography code

Reply:
Created variables for values which are being used at different places in the code in the same logical sense. 
Left few values as they are, since their usages are self-explanatory.